anime-relations.txt

```diff
@@ -41,6 +41,9 @@
 # Acchi Kocchi (TV) -> ~: Place=Princess
 - 12291|6701:13 -> 16273|7401:1
 
+# Ai Mai Mi: Mousou Catastrophe -> ~ Special
```

Comment:
Should be `Choboraunyopomi Gekijou Dai Ni Maku Ai Mai Mii: Mousou Catastrophe -> ~ Special`. Then, the rule should be reordered.